source/_integrations/airos.markdown

```diff
@@ -35,18 +36,50 @@ This integration only supports devices running airOS 8 and already configured us
 
 {% include integrations/config_flow.md %}
 
+## Operating roles
+
+Depending on the device's placement, it will be configured as either an 'Access Point' (AP) acting as the central device or a 'Station' connecting as a client. An AP can have multiple stations connected to it, whereas a station will only connect to a single AP.
+
+For stations in particular, they can operate in either a 'Bridge' or 'Router' role:
+
+- In 'Bridge' mode, which is the default and most common configuration, the airOS device simply bridges the wireless and Ethernet connections. In simplistic terms, it functions as a transparent network cable, making it invisible to the devices on both the station and AP sides.
+
+- In 'Router' mode, the airOS device acts as a small router, performing Network Address Translation (NAT) as well as providing DHCP services for devices connected to its LAN port. The airOS device in this mode can also operate as a PPPoE client, authenticating with a central PPPoE server to receive its IP address, gateway, and other network settings.
```

Comment:
```suggestion
- In 'Router' mode, the airOS device acts as a small router, performing Network Address Translation (NAT) as well as providing <abbr title="Dynamic Host Configuration Protocol">DHCP</abbr> services for devices connected to its <abbr title="Local Area Network">LAN</abbr> port. The airOS device in this mode can also operate as a <abbr title="Point-to-Point Protocol over Ethernet">PPPoE</abbr> client, authenticating with a central <abbr title="Point-to-Point Protocol over Ethernet">PPPoE</abbr> server to receive its IP address, gateway, and other network settings.
```
I realize that these acronyms are very common in networking. And that by spelling out the acronym, people who are new to networking still don't have enough context. But it might help a little bit.